website/docs/docs/cloud-integrations/configure-auto-exposures.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Auto-exposures"
+id: "configure-auto-exposures"
+sidebar_label: "Configure auto-exposures"
+description: "Import and auto-generate exposures from dashboards and understand how models are used in downstream tools for a richer lineage."
+image: /img/docs/cloud-integrations/auto-exposures/explorer-lineage2.jpg
+---
+
+# Configure auto-exposures <Lifecycle status='beta' />
+
+As a data team, it’s critical that you have context into the downstream use cases and users of your data products. [Auto-exposures](/docs/collaborate/auto-exposures) integrates natively with Tableau (Power BI coming soon) and [auto-generates downstream lineage](/docs/collaborate/auto-exposures#view-auto-exposures-in-dbt-explorer) in dbt Explorer for a richer experience.
```

Comment:
Could we axe the power BI coming soon callout? It is next, but I'd rather not say this publicly in our docs before we start development on it and we don't have a timeline of feasibility.

Reply:
Yep! Resolved in https://github.com/dbt-labs/docs.getdbt.com/pull/5896/commits/84fe9bdad242ebd79d16a482e7b5f711d7d2e466